### PR TITLE
TigerGraph CoPilot Support

### PR DESCRIPTION
**Pull request by parkererickson-tg**
Thank you for contributing to LangChain!

- [ X] **TIgerGraph CoPilot Support**: "community: add TigerGraph vector store support, graph modifications"

    - **Description:** adds support for TigerGraph CoPilot vector store functionality and updates the graph store. 
    - **Dependencies:** none.
    - **Twitter handle:**: @tigergraphdb

- [ X] **Add tests and docs**

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, hwchase17.

### CODE REVIEW

File: libs/community/langchain_community/vectorstores/tigergraph.py [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    Used in conjunction with TigerGraph CoPilot SupportAI functionality.
+    For more information, see https://docs.tigergraph.com/tg-copilot/current/intro/.
+
+    To use, you should have the ``pyTigerGraph`` python package installed.
+
+    Args:
+
+
+    Example:
+        .. code-block:: python
```
Comment (baskaryan) [inline libs/community/langchain_community/vectorstores/tigergraph.py:34]: fill in

File: libs/community/langchain_community/vectorstores/tigergraph.py [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+                )
+                content_upserted += self.conn.upsertVertices(
+                    "Content", content_vertices[i : i + upsert_size]
+                )
+
+        return {
+            "documents_upserted": documents_upserted,
+            "has_content_edges_upserted": has_content_edges_upserted,
+            "content_upserted": content_upserted,
+        }
```
Comment (baskaryan) [inline libs/community/langchain_community/vectorstores/tigergraph.py:157]: should return list of string ids for the inserted docs

File: libs/community/langchain_community/vectorstores/tigergraph.py [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+    def add_documents_from_blob(
+        self,
+        data_source: str,
+        data_source_config: Dict[str, Any],
+        loader_config: Dict[str, Any],
+        data_path: str,
+        file_format: str = "json",
+    ):
```
Comment (baskaryan) [inline libs/community/langchain_community/vectorstores/tigergraph.py:166]: return type

**Review by baskaryan — COMMENTED** [review]
can we add a docs page to docs/docs/integrations/vectorstores

File: libs/community/langchain_community/vectorstores/tigergraph.py (hunk 1 above)
Comment (parkererickson-tg) [reply libs/community/langchain_community/vectorstores/tigergraph.py]: Thanks for the catch

**Comment by parkererickson-tg** [comment]
Added docs. Thanks for reviewing @baskaryan !

**Pull request closed without merging**